cookbook/migration-v1-v2.md

```diff
@@ -1,45 +1,42 @@
-# Aún no está traducido...
----
+# Migración desde 0.x (v1) a v2 {#migrating-from-0-x-v1-to-v2}
 
-# Migrating from 0.x (v1) to v2
+¡A partir de la versión `2.0.0-rc.4`, pinia soporta tanto Vue 2 como Vue 3! Esto significa que todas las nuevas actualizaciones se aplicarán a esta versión 2 para que tanto los usuarios de Vue 2 como los de Vue 3 puedan beneficiarse de ella. Si estás usando Vue 3, esto no cambia nada para ti ya que ya estabas usando el rc y puedes revisar [CHANGELOG](https://github.com/vuejs/pinia/blob/v2/packages/pinia/CHANGELOG.md) para una explicación detallada de todo lo que cambió. ¡Si no, **esta guía es para ti**!
 
-Starting at version `2.0.0-rc.4`, pinia supports both Vue 2 and Vue 3! This means, all new updates will be applied to this version 2 so both Vue 2 and Vue 3 users can benefit from it. If you are using Vue 3, this doesn't change anything for you as you were already using the rc and you can check [the CHANGELOG](https://github.com/vuejs/pinia/blob/v2/packages/pinia/CHANGELOG.md) for a detailed explanation of everything that changed. Otherwise, **this guide is for you**!
+## Depreciaciones {#deprecations}
```

Comment:
This could also be translated as "Características Obsoletas" because here in Spain at least we don't use the term "depreciación" to refer to something that is deprecated. What do you think? Also if you are agree please look for the same word and change it.

Reply:
you're right, done